msteams-platform/TOC.yml

```diff
@@ -491,6 +491,12 @@
   - name: Overview
     href: messaging-extensions/what-are-messaging-extensions.md
     displayName: handleTeamsMessagingExtensionFetchTask
+  - name: API based message extension
+    href: messaging-extensions/build-api-based-message-extension.md
+    displayName: OpenAPI specification, Search, single parameter, search based message extension
+  - name: Bot based message extension
```

Comment:
I know naming isn't finalized, but if these working terms do become official, should they be hyphenized?

API-based message extension
Bot-based message extension

Reply:
Updated API-based message extension and Bot-based message extension